ClangSharp/Generated.cs

```diff
@@ -3,294 +3,6 @@ namespace ClangSharp
     using System;
     using System.Runtime.InteropServices;
 
-    public partial struct CXTargetInfoImpl
```

Comment:
The changes here are a side effect of doing two passes. I validated manually there was zero diff without that commit.